quarkus-modules/quarkus-kogito/src/main/java/com/baeldung/kogito/rules/LoanApplicationService.java

```diff
@@ -0,0 +1,41 @@
+package com.baeldung.kogito.rules;
+
+import com.baeldung.kogito.rules.model.LoanApplication;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+import org.kie.kogito.incubation.application.AppRoot;
+import org.kie.kogito.incubation.common.MapDataContext;
+import org.kie.kogito.incubation.rules.RuleUnitIds;
+import org.kie.kogito.incubation.rules.services.RuleUnitService;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+@ApplicationScoped
+public class LoanApplicationService {
+
+    @Inject
+    AppRoot appRoot;
+    @Inject
+    RuleUnitService ruleUnitService;
+
+    public Stream<LoanApplication> evaluate(LoanApplication... applications) {
+        var queryId = appRoot
+                .get(RuleUnitIds.class)
+                .get(LoanApplicationUnit.class)
+                .queries()
+                .get("applications");
+        var ctx = MapDataContext
+                .of(
+                        Map.of("applications", applications)
+                );
+        return ruleUnitService
+                .evaluate(queryId, ctx)
+                .map(
+                        result -> result
+                                .as(MapDataContext.class)
+                                .get("$result", LoanApplication.class)
+                );
```

Comment:
Indentation should be be four spaces per level, with continuation lines needing only two, and fluent-style API calls OK with either two or four (though four is more standard).